sympy/functions/special/tests/test_spec_polynomials.py

```diff
@@ -150,7 +151,7 @@ def test_legendre():
         -sqrt(Rational(3, 7) + Rational(2, 35)*sqrt(30)): 1,
     }
 
-    n = Symbol("n")
+    n = Symbol("n", integer=True)
```

Comment:
Is this change needed for the subsequent tests to work?

Reply:
Not really. I probably just though that it made more sense to declare the polynomial order to be integer... Trying to add as much information as possible when using SymPy so it got passed on to tests.